test/functional/apps/dashboard/group1/dashboard_unsaved_listing.ts

```diff
@@ -86,15 +86,18 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('shows a warning on create new, and restores panels if continue is selected', async () => {
-      await PageObjects.dashboard.clickNewDashboardExpectWarning(true);
+      await PageObjects.dashboard.clickNewDashboard({ continueEditing: true, expectWarning: true });
```

Comment:
Looks like this was the only use of `clickNewDashboardExpectWarning` - can you remove it from the Dashboard page object to further clean this up? :eyes:

Reply:
it was removed https://github.com/elastic/kibana/pull/179560/files#diff-d38ed9c9171044b875ae8172816be43bc3518eda75efe158d7a455a87fd296cbL386-L406